darwin/dataset/remote_dataset.py

```diff
@@ -228,7 +230,7 @@ def pull(
         if release is None:
             release = self.get_release()
 
-        if release.format != "json":
+        if release.format != "json" and release.format != "darwin_json_2":
```

Comment:
Again, if we're going for consistency they should both be `darwin_json_v1` and `darwin_json_v2` or `darwin_v1` and `darwin_v2` but not a mix

Reply:
We can't change old format name. This is exposed in our APIs, this wouldn't be backwards compatible change. In V2 APIs the old format will be eventually dropped entirely, but we cannot rename it right now. 
I agree that consistency would be cool, I just don't see an option to do it.